docker.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export NAME=osr_django
+export PORTS="-p 8000:8000"
+export VOLUME="-v $(pwd):/app"
+
+docker stop $NAME
+sleep 1
```

Comment:
what are these `sleep`s for?

Reply:
it takes a little bit of time for the docker backend to carry out the order, the sleep 1 guarantees that the order is done before sending out the next one.

Reply:
`docker stop` doesn't exit until the container is stopped. you can confirm by running `docker run --name stoptest -it python:3.5-stretch` and then `docker stop stoptest` in a separate shell. it won't exit until you exit the python shell

Reply:
It's a standard where I work to add the sleep command between two docker commands because one time something went wrong and ever since then we've added those. You can delete them, I'm just trying to be safe